src/main/studio/ui/SettingsDialog.java

```diff
@@ -219,6 +229,13 @@ private void initComponents() {
         comboBoxLineEnding = new JComboBox<>(LineEnding.values());
         comboBoxLineEnding.setSelectedItem(Config.getInstance().getEnum(Config.DEFAULT_LINE_ENDING));
 
+        JLabel lblFontSize = new JLabel("Font:");
+        spnFontSize = new JSpinner(new SpinnerNumberModel(Config.getInstance().getFontSize(), 8, 72, 1));
+
+        GraphicsEnvironment ge = GraphicsEnvironment.getLocalGraphicsEnvironment();
+        cbFontName = new JComboBox(ge.getAvailableFontFamilyNames());
+        cbFontName.getModel().setSelectedItem(Config.getInstance().getFontName());
+
```

Comment:
I thought to use fontchooser from org.drjekyll (which is already added as a dependency in the build.gradle).